packages/cms-lib/lib/config.js

```diff
@@ -13,10 +13,23 @@ const {
   DEFAULT_HUBSPOT_CONFIG_YAML_FILE_NAME,
   EMPTY_CONFIG_FILE_CONTENTS,
   Mode,
+  API_KEY_AUTH_METHOD,
+  OAUTH_AUTH_METHOD,
+  PERSONAL_ACCESS_KEY_AUTH_METHOD,
+  OAUTH_SCOPES,
+  ENVIRONMENT_VARIABLES: {
+    HUBSPOT_REFRESH_TOKEN,
+    HUBSPOT_API_KEY,
+    HUBSPOT_CLIENT_ID,
+    HUBSPOT_CLIENT_SECRET,
+    HUBSPOT_PERSONAL_ACCESS_KEY,
+    HUBSPOT_PORTAL_ID,
+  },
```

Comment:
Our constants file is getting heavy.  At some point we should consider managing enums separately.

Instead of nested de-structuring here, how about we export an `EnvironmentVariables` enum that has these values.